public/src/client/topic.js

```diff
@@ -143,7 +143,7 @@ define('forum/topic', [
 			if (components.get('post/anchor', postIndex).length) {
 				return navigator.scrollToPostIndex(postIndex, true);
 			}
-		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > 5) {
+		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > ajaxify.data.config.bookmarkThreshold) {
```

Comment:
[`ajaxify.js` seems to suggest the opposite, by _explicitly setting_ `config` in `ajaxify.data` on completion...](https://github.com/NodeBB/NodeBB/blob/master/public/src/ajaxify.js#L246)